.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,49 @@
+name: "Bug report 🐛"
+description: "Create a report to help us fix something that is currently broken."
+title: "[BUG] - <title>"
+labels: ["type: bug 🐛"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting!
+        Before submitting a bug, please make sure the issue hasn't already been addressed by searching through
+        the past issues in this repository.
+
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: |
+        A clear and concise description of what the bug is.
+        Precisely describe what you expected to happen and what actually happened.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How to reproduce the bug?
+      description: |
+        Provide a minimal code example and detailed step-by-step guidelines to reproduce the error.
+    validations:
+      required: false
```

Let's make this required if we already have a template. Bug report without repro is hard to act on. Worst case, someone could just put "N/A" or the like in there.

```suggestion
      required: true
```